### PR TITLE
Delivery ancestor rapp and some bugfixes

### PR DESCRIPTION
**Pull request by stonier**
- ~~First iteration of delivery ancestor rapp (public interface/params just stubs)~~
- Bugfix to rocon app manager - was a broken assumption on current_rapp not being None
- Bugfix to rocon app utilities - moved variable use inside the scope of the try block before the excepts.
- Bugfix base name conversion so proper ros naming rules apply when not using uuids.
- Removed runtime information from the list_rapps topic (should all be in 'status').
- Syntax highlighting for the ugly/hard to read `rocon_app` command.

### CODE REVIEW

File: rocon_app_manager/src/rocon_app_manager/rapp.py [hunk 1]
```diff
@@ -29,6 +29,7 @@ class Rapp(object):
         Got many inspiration and imported from willow_app_manager implementation
     '''
     # I should add a __slots__ definition here to make it easy to read
+    # Dont think so : slots prevent using dict and other high level python stuff. (AlexV)
```
Comment (stonier) [inline rocon_app_manager/src/rocon_app_manager/rapp.py]: Aye, learned this after I wrote that comment a long time ago.

File: rocon_apps/apps/delivery/delivery.interface [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+# publisher `robot_status` should eventually be moved out and published by something generic (e.g.
+# robot app manager) rather than this rapp itself. 
+
+publishers: 
+  - name: homebase
+    type: std_msgs/String
+  - name: pose
+    type: geometry_msgs/PoseStamped
+  - name: robot_status
+    type: gopher_std_msgs/Status
```
Comment (jihoonl) [inline rocon_apps/apps/delivery/delivery.interface]: where do I find this message format?

**Comment by jihoonl** [comment]
All looks good except delivery rapp. I would like to see if this delivery rapp can be used for other robots like waiterbot and also gopher free.

Can you provide any pointer that describes how this interface is designed? and pointer to gopher_std_msgs repository?

**Comment by stonier** [comment]
Oh, good point. Can't have this here if it's pointing to `gopher_std_msgs`.

The public interface is a work in progress. Looks like we have two options:
1. drop this rapp from here for now OR
2. move the supporting messages into rocon_msgs somewhere (more work but easier for collaborating on). 

What would you prefer?

**Comment by jihoonl** [comment]
I would prefer to drop this rapp from this PR. and continue the rapp interface discussion in separate thread. How about this?

Having delivery meta rapp sounds good since we need for other service demos using waiterbot and turtlebot.

**Comment by stonier** [comment]
Moved out the deliver rapp. Added you to the `gopher_concert` repo so you can see where this implementation is going and provide feedback on issues there.

[Delivery Rapp Thread](https://bitbucket.org/yujinrobot/gopher_concert/issue/1/delivery-rapp-specification).

**Comment by jihoonl** [comment]
:+1:

**Pull request merged**